src/js/angular/utils/repositories-utils.service.js

```diff
@@ -0,0 +1,25 @@
+const modules = [];
+
+angular
+    .module('graphdb.framework.repositories', modules)
+    .factory('RepositoriesUtilService', RepositoriesUtilsService);
+
+RepositoriesUtilsService.$inject = ['RDF4JRepositoriesRestService'];
+
+function RepositoriesUtilsService(RDF4JRepositoriesRestService) {
```

Comment:
Didn't we untangled the cyclic dependency between the services which enforsed you to write down this redundant utility service?

Reply:
I will check this.

Reply:
Yes, the feature was moved to the repositories service. I just forgot to delete this.

Reply:
Done